packages/blade/src/components/Card/__tests__/CardInteractive.native.test.tsx

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/no-empty-function */
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import React from 'react';
+import { fireEvent, waitFor } from '@testing-library/react-native';
+import { Linking } from 'react-native';
+import { CardBody, Card } from '..';
+import renderWithTheme from '~utils/testing/renderWithTheme.native';
+import { Box } from '~components/Box';
+import { Text } from '~components/Typography';
+import { Button } from '~components/Button';
+
+describe('<Card />', () => {
+  it('should render a interactive Card', () => {
+    const { toJSON } = renderWithTheme(
+      <Card onClick={() => {}} scaleOnHover isSelected={true}>
+        <CardBody>Plain Card</CardBody>
+      </Card>,
+    );
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render a clickable Card', () => {
+    const { toJSON } = renderWithTheme(
+      <Card onClick={() => {}}>
```

Comment:
we already have better story with event handlers for onclick so probably remove this?

Reply:
Better to have atomic tests easier to debug if anything breaks.